specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -660,6 +660,46 @@
         }
       }
     },
+    "EnvironmentSkuProperties": {
+      "description": "Managed Environment resource SKU properties.",
```

Comment:
seems this definition is in middle, could we move down?

Reply:
Perhaps we should make this alphabetical eventually